org.openmw.OpenMW.json

```diff
@@ -0,0 +1,265 @@
+{
+  "app-id": "org.openmw.OpenMW",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "openmw-launcher",
+  "rename-appdata-file": "openmw.appdata.xml",
+  "rename-desktop-file": "openmw.desktop",
+  "finish-args": [
+    "--share=ipc", "--socket=x11", "--device=dri",
+    "--socket=pulseaudio"
+  ],
+  "build-options" : {
+    "cflags": "-O2 -g",
+    "cxxflags": "-O2 -g"
+  },
+  "cleanup": ["/include", "/lib/pkgconfig",
+    "/share/pkgconfig", "/share/aclocal",
+    "/share/doc",
+    "/man", "/share/man", "/share/gtk-doc",
+    "/share/vala",
+    "*.la", "*.a"],
+    "modules": [
+      "shared-modules/lame/lame-3.99.5.json",
+      {
+        "name": "ffmpeg",
+        "config-opts": [
+          "--enable-gpl",
+          "--enable-libmp3lame",
+          "--enable-libtheora",
+          "--enable-libvorbis",
+          "--enable-nonfree",
+          "--enable-shared",
+          "--disable-static",
+          "--disable-programs",
+          "--disable-doc",
+          "--disable-avdevice",
+          "--disable-avfilter",
+          "--disable-postproc"
+        ],
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://ffmpeg.org/releases/ffmpeg-3.2.4.tar.xz",
+            "sha256": "6e38ff14f080c98b58cf5967573501b8cb586e3a173b591f3807d8f0660daf7a"
+          }
+        ]
+      },
+      {
+        "name": "qt4",
```

Comment:
Qt4 is in `shared-modules`.

Reply:
This requires the QtOpengl module, which is explicitly disabled in the shared module, so should it be activated there or done separately here?

Reply:
Hmm, it is fine to keep it here.